include/picongpu/param/atomicPhysics.param

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2019 Brian Marre
```

Comment:
Here and other files, please make sure the end year is 2020.

Reply:
okay, I am on it

Reply:
done

Reply:
Thanks!